desc/coils.py

```diff
@@ -1205,7 +1214,7 @@ def from_symmetry(cls, coils, NFP=1, sym=False):
             rotated_coils.rotate(axis=[0, 0, 1], angle=2 * jnp.pi * k / NFP)
             coilset += rotated_coils
 
-        return cls(*coilset)
+        return cls(*coilset, check_intersection=True)
```

Comment:
This should be `check_intersection=check_intersection`, otherwise there is no point in checking if any coils lie on the symmetry plane. But as of now, `check_intersection=True` is the default in `CoilSet.__init__` so this isn't actually necessary anyways...

Reply:
True